VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -269,12 +268,12 @@ private void updateUIForOrientation(){
             setActionBarVisible(false);
             findViewById(R.id.course_unit_nav_bar).setVisibility(View.GONE);
             pager.setPagingEnabled(false);
-            environment.getSegment().trackCourseComponentViewed(selectedUnit.getId(), courseData.getCourse().getId(), false);
+            environment.getSegment().trackCourseComponentViewed(selectedUnit.getId(), courseData.getCourse().getId());
         } else {
             setActionBarVisible(true);
             findViewById(R.id.course_unit_nav_bar).setVisibility(View.VISIBLE);
             pager.setPagingEnabled(true);
-            environment.getSegment().trackCourseComponentViewed(selectedUnit.getId(), courseData.getCourse().getId(), true);
+            environment.getSegment().trackCourseComponentViewed(selectedUnit.getId(), courseData.getCourse().getId());
```

Comment:
Since these two calls are identical now, we can bring them outside the `if` `else` statement.